From: James Rowe <jnrowe@gmail.com>
Subject: Removed unused intersphinx settings.
Status: LOCAL

These unused settings cause each doc build to pull large remote resources.

diff --git a/docs/conf.py b/docs/conf.py
index 821b47c..9a6de44 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -217,7 +217,5 @@ man_pages = [
 
 
 intersphinx_mapping = {
-    'python': ('http://python.readthedocs.org/en/latest/', None),
-    'django': ('http://django.readthedocs.org/en/latest/', None),
-    'sphinx': ('http://sphinx.readthedocs.org/en/latest/', None),
+    'python': ('http://python.readthedocs.org/en/latest/', None)
 }
